example/example-metadata.ttl

```diff
@@ -34,39 +34,150 @@
+        ]
+    ].
+
+
+<beol> dsp-repo:hasSpatialCoverage [
+        a schema:Place ; 
+        schema:url [
+            a schema:URL ;
+            schema:propertyID [
+                a schema:PropertyValue ;
+                schema:propertyID "Geonames" ;
+            ] ;
+            schema:url "https://www.geonames.org/3175395/italian-republic.html" ; 
+        ]
+    ].
+
+<beol> dsp-repo:hasSpatialCoverage [
+        a schema:Place ; 
+        schema:url [
+            a schema:URL ;
+            schema:propertyID [
+                a schema:PropertyValue ;
+                schema:propertyID "Geonames" ;
+            ] ;
+            schema:url "https://www.geonames.org/6269131/england.html" ; 
+        ]
+    ].
+
+
+# <dietrich> dsp-repo:hasDiscipline [
+#    a schema:URL ;
+#    schema:propertyID [
+#        a schema:PropertyValue ;
+#        schema:propertyID "SKOS UNESCO Nomenclature" ;
+#    ] .
+#    schema:url "http://skos.um.es/unesco6/11" ;
+# ] .
+
+<beol> dsp-repo:hasFunder <beol-funder> . 
 <beol> dsp-repo:hasGrant <beol-grant> .
-<beol> dsp-repo:hasURL "https://beol.dasch.swiss/" . 
-<beol> dsp-repo:hasDateCreated "09.2017" . 
-<beol> dsp-repo:hasDateModified "04.2020" . 
+
+<beol> dsp-repo:hasURL "https://beol.dasch.swiss/" .
+
 <beol> dsp-repo:hasShortcode "0801" . 
 <beol> dsp-repo:hasAlternateName "beol" .
+<beol> dsp-repo:hasDataManagementPlan <beol-DMP> .
+<beol> dsp-repo:hasContactPoint  [
+        a schema:ContactPoint ; 
+        schema:url [
+            a schema:URL ;
+            schema:propertyID [
+                a schema:PropertyValue ;
+                schema:propertyID "Contact Point" ;
+            ] ;
+            schema:url "https://beol.dasch.swiss/" ; 
+        ]
+    ].
 
 <beol-dataset> rdf:type dsp-repo:Dataset .
 <beol-dataset> dsp-repo:hasIdentifier "http://ark.dasch.swiss/ark:/72163/1/0801" . 
+<beol-dataset> schema:sameAs "https://beol.dasch.swiss/" .
+<beol-dataset> dsp-repo:hasTitle "Bernoulli-Euler Online" .
+<beol-dataset> dsp-repo:hasAlternativeTitle "BEOL" .
 <beol-dataset> dsp-repo:hasAbstract "The Bernoulli-Euler Online (BEOL) project is a research platform for the study of early modern mathematics and science. This Beta release integrates the two edition projects Basler Edition der Bernoulli-Briefwechsel (BEBB) and Leonhardi Euleri Opera Omnia (LEOO) IVA/IV into one platform. As a next step, Jacob Bernoulli's scientific notebook Meditationes will be also integrated in this platform. Currently BEOL is connected to the repositories of The Newton Project and the Briefportal Leibniz initiating the formation of a network of digital editions of the correspondence among the early modern mathematicians." .
 <beol-dataset> dsp-repo:hasEncodingFormat "text" . 
 <beol-dataset> dsp-repo:hasEncodingFormat "image" . 
-<beol-dataset> dsp-repo:hasDocumentation "Work in progress" . 
+<beol-dataset> dsp-repo:hasDocumentation "Work in progress" .
+<beol-dataset> dsp-repo:hasLicense "CC BY Test" .
```

Comment:
As far as I'm aware, BEOL doesn't actually state anywhere under what license it's published.  
(This has been a point of critique brought forward by the BEBB team at one point.)  
So either this information is wrong, or it's good that it's finally made public.

Also, what does "Test" mean there? I'm guessing this should be `CC BY`